Makefile

```diff
@@ -326,12 +326,14 @@ create-local-identity: id_dir=/tmp/
 create-local-identity: id_password=local
 create-local-identity: id_prefix=.identity-local_
 create-local-identity: ## run HOPRd from local repo
-	ETHERSCAN_API_KEY="" IDENTITY_PASSWORD="${id_password}" \
+	if [ ! -f "${id_dir}${id_prefix}0.id" ]; then \
+		ETHERSCAN_API_KEY="anykey" IDENTITY_PASSWORD="${id_password}" \
```

Comment:
are those `anykey` intentional ? Looks like a testing value

Reply:
Any value can be there... it does not matter when creating local identities. `hopli` just needs that env to be set.